lib/nest-thermostat-accessory.js

```diff
@@ -213,7 +223,7 @@ NestThermostatAccessory.prototype.setTargetTemperature = function (targetTempera
 	return this.cancelAutoAway()
 		.then(this.updateDevicePropertyAsync.bind(this, key, targetTemperature, prop + "target temperature"))
 		.asCallback(callback);
-};
+}, 5000);
```

Comment:
A 5s delay is right on the cusp of making me question if I actually completed an action. Consider making this user-configurable. Alternatively, decrease the delay, perhaps to 2 or 3s.